### ISSUE TITLE
Thread 1: EXC_BAD_ACCESS (code=1, address=0x0)

### ISSUE THREAD

**Issue by yuntan** [nalexn/ViewInspector#243]
```
let view = ContentView()
try view.inspect().vStack() // -> crash
```

`Thread 1: EXC_BAD_ACCESS (code=1, address=0x6)` happens at Inspector.swift line 93. version 0.9.5.

<img width="1440" alt="スクリーンショット 2023-05-05 16 25 20" src="https://user-images.githubusercontent.com/574575/236399298-403405eb-8587-4afa-b638-defba8b0e99d.png">

minimal `ContentView`:

```
struct ContentView: View {
    let tag = ContentView.self
    
    var body: some View {
        VStack {
            Image(systemName: "globe")
                .imageScale(.large)
                .foregroundColor(.accentColor)
            Text("Hello, world!")
        }
        .padding()
    }
}
```

`let tag = ContentView.self` is cause of this problem.

**Comment by nalexn** [comment]
Please provide a minimal code with which the issue can be reproduced; just two lines aren't helpful

**Comment by yuntan** [comment]
@nalexn updated.

**Comment by nalexn** [comment]
Released in v0.9.8
